### DISCUSSION TITLE
How to contribute (WIP)

### DISCUSSION THREAD

**Discussion by FilipePfluck** [chakra-ui/ark#2084] [General]
### Description

the contributing.md linked in the readme seems to be missing: 
![Screenshot from 2024-01-15 16-57-25](https://github.com/chakra-ui/ark/assets/62773200/00355047-0aba-4f20-97b8-de231716dde2)

I also forked and cloned the repo and searched for any file similar to contributing.md and it seems there is none. 

### Link to Reproduction (or Detailed Explanation)

https://github.com/chakra-ui/ark/blob/main/CONTRIBUTING.md

### Steps to Reproduce

1. open the link

### Ark UI Version

1.0.0

### Framework

- [ ] React
- [ ] Solid
- [ ] Vue

**Comment by Omikorin** [comment]
Thanks, I've renamed contributing to code of conduct because it was code of conduct. We need to write some contributing, but I didn't decide to use generator or write a simple instruction for dev env

**Comment by FilipePfluck** [comment]
I see, that makes sense. I cloned the repo and was looking for instructions to get the project running. Let me know when the contributing file is ready.

**Comment by Omikorin** [comment]
For now, look [at this discussion](https://github.com/chakra-ui/ark/discussions/1941#discussioncomment-7982825)
